### ISSUE TITLE
I don't think this $ARCH is used anywhere, am I wrong?

### ISSUE THREAD

**Issue by hpux735** [uraimo/buildSwiftOnARM#27]
https://github.com/uraimo/buildSwiftOnARM/blob/6e610375bb18290f0f0a456f9e3054ece92bb2da/build.sh#L6

**Comment by uraimo** [comment]
Correct, we were using it to fix the install directory for some header, not needed anymore, fixing it.
